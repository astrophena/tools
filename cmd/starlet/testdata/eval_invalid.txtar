update: start
-- bot.star --
def handle(update):
    # Should be an error (invalid code).
    res = eval("prnt(greeting", environ = {
        "greeting": "Hello, world!",
    })
