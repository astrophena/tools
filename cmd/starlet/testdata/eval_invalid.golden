[
  {
    "method": "sendMessage",
    "args": {
      "chat_id": 123456789,
      "entities": [
        {
          "length": 118,
          "offset": 25,
          "type": "pre"
        }
      ],
      "link_preview_options": {
        "is_disabled": true
      },
      "text": "‚ùå Something went wrong:\n\nTraceback (most recent call last):\n  //bot.star:3:15: in handle\nError in eval: \u003ceval\u003e:1:14: got end of file, want ')'\n\n"
    }
  }
]